Tests/Engine/Settings.tests.ps1

```diff
@@ -218,6 +218,11 @@ Describe "Settings Class" {
                 @{ Expr = '@("A","B","C")'; Count = 3 }
                 @{ Expr = '@()'; Count = 0 }
                 @{ Expr = '@(7)'; Count = 1 }
+                @{ Expr = "'1',`n'2',`n'3'"; Count = 3 }
+                @{ Expr = "@(1`n3`n5`n7)"; Count = 4 }
+                @{ Expr = "'1',`r`n'2',`r`n'3'"; Count = 3 }
+                @{ Expr = "@(1`r`n3`r`n5`r`n7)"; Count = 4 }
+                @{ Expr = "@(1,2,3`n7,8,9)"; Count = 6 }
```

Comment:
i wonder if we should have a test which actually uses a settings file (since that's how you found this)

Reply:
Hmm, 3 of those added tests fail when being run against the current `development` branch, so they seem to be useful on their own. Maybe a simple additional test for files would be good as well if there is a difference in terms of parsing?

Reply:
Added the test below in a new file